Add MMC power sequence for usdhc2.
Add bluetooth node to uart3.

--- a/arch/arm/boot/dts/imx6qdl-wandboard-revc1.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-wandboard-revc1.dtsi
@@ -6,6 +6,15 @@
 
 #include "imx6qdl-wandboard.dtsi"
 
+/ {
+	pwrseq_usdhc2: usdhc2pwrseq {
+		compatible = "mmc-pwrseq-simple";
+		reset-gpios = <&gpio1 26 GPIO_ACTIVE_LOW>,	/* WL_REG_ON */
+			      <&gpio5 30 GPIO_ACTIVE_LOW>,	/* BT_REG_ON */
+			      <&gpio5 21 GPIO_ACTIVE_LOW>;	/* BT_RST_N */
+	};
+};
+
 &iomuxc {
 	pinctrl-0 = <&pinctrl_hog>;
 
@@ -28,8 +37,15 @@
 	};
 };
 
+&uart3 {
+	bluetooth {
+		compatible = "brcm,bcm4330-bt";
+	};
+};
+
 &usdhc2 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_usdhc2>;
+	mmc-pwrseq = <&pwrseq_usdhc2>;
 	status = "okay";
 };
